agent/functional_tests/testdata/taskdefinitions/mdservice-validator-unix/task-definition.json

```diff
@@ -0,0 +1,10 @@
+{
+  "family": "ecsftest-mdservice-validator",
+  "containerDefinitions": [{
+    "image": "127.0.0.1:51670/busybox:latest",
+    "name": "mdservice-validator-unix",
+    "cpu": 10,
+    "memory": 10,
+    "command": ["sh", "-c", "sleep 1 && if cat $ECS_CONTAINER_METADATA_FILE | grep \"READY\"; then exit 42; else exit 1; fi"]
```

Comment:
`sleep 1` may not always be sufficient. Can you increase this duration to something like `10s`? Or, you can check for the existence in a loop and poll on the file for a maximum of `1m`.